english/doug-adams/arthur-dent.md

```diff
@@ -0,0 +1,11 @@
+You find yourself, still dressed in pajamas and robe, lying in the dirt in
+front of a bulldozer, blocking it from destroying your house.
+
+This must be Thursday. You never could get the hang of Thursdays.
+
+You friend Ford appears.
+
+"C'mon. Let's go to the [Pub](./ford-prefect.md). He said they
+won't destroy your house until they've finished the beers." Ford says.
+
+Simply motion to the [bulldozer](../marshmallows.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.